apps/juicebox_web/assets/js/videos/video-page.js

```diff
@@ -2,28 +2,46 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import Search from '../search/search';
 import Reactions from '../reactions/reactions';
-import { addVideo, toggleExpanded } from './actions';
+import { addVideo, toggleExpanded, voteDown, voteUp } from './actions';
 import Video from './video';
 import VideoList from './video-list';
 import styles from './video-page.scss';
 
 class VideoPage extends Component {
   render() {
-    const { queue, playing, playingStartTime, expanded, toggleExpanded } = this.props;
+    const {
+      expanded,
+      playing,
+      playingStartTime,
+      queue,
+      streamId,
+      toggleExpanded,
+      votes,
+      voteDown,
+      voteUp,
+    } = this.props;
+
+    console.log('votes', votes);
```

Comment:
Unexpected console statement.

[Read more about it here.](http://eslint.org/docs/rules/no-console)